jck/diff/diff.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash 
+
+setup()
+{
+	cd $WORKDIR
+	version=$1
+	repo=$2
+	
+	if [ -d "$WORKDIR/$version" ] ; then 
+		echo "Using existing test repo at: $WORKDIR/$version"
+	else 
+		echo "Git cloning test materials from $repo..."
+		git clone $repo $version
```

Comment:
Please use `git clone --depth 1 -q ...` as we do not need git history and download progress.

Reply:
Updated. Thanks for pointing out!